containers/deploy/foreman/deployments.yml

```diff
@@ -32,7 +32,7 @@
     - stop
     - restart
 
-- name: Create deployment, and scale replicas up
+- name: Create Foreman deployment
```

Comment:
WFIW these do the create as well as the delete, we should probably name them a bit better.

Reply:
That's fair. This was mostly inspired by being annoyed that I couldn't tell what stage it was at cause everything had the same name.

Reply:
Yeah, definitely understand the change you made, just seeing the diff here made me remember about the create/delete thing.